pkg/networkserver/networkserver_util_internal_test.go

```diff
@@ -1200,7 +1200,9 @@ func (env TestEnvironment) AssertScheduleJoinAccept(ctx context.Context, dev *tt
 					events.WithIdentifiers(&dev.EndDeviceIdentifiers),
 				).New(ctx),
 				EvtScheduleJoinAcceptSuccess.With(
-					events.WithData(&ttnpb.ScheduleDownlinkResponse{}),
+					events.WithData(&ttnpb.ScheduleDownlinkResponse{
+						Delay: ttnpb.ProtoDurationPtr(0),
+					}),
```

Comment:
For successful downlinks, this cannot be empty. So we get a zero value here and below.